src/main/java/seedu/address/model/Model.java

```diff
@@ -23,6 +24,9 @@
     /** Deletes the given person. */
     void deletePerson(ReadOnlyPerson target) throws PersonNotFoundException;
 
+    /** Deletes the given tag. */
```

Comment:
header comment is obvious. no header comment is necessary if the method is obvious. its okay to not have a header comment